ci/README.md

```diff
@@ -363,11 +363,8 @@ This was deemed to be error prone and not worth the effort.
 
 This script checks and fixes issues with `.gitignore` rules:
 
-1. Makes sure git ignores the `.sql` files and expected output files that are generated
-   from `.source` template files. If you created or deleted a `.source` file in a commit,
-   git ignore rules should be updated to reflect this change.
 
-2. Makes sure we do not commit any generated files that should be ignored. If there is an
+1. Makes sure we do not commit any generated files that should be ignored. If there is an
```

Comment:
Does it make sense to have a numbered list if we have only one item left? I am not so sure.

Reply:
I'm fine either way